CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 # Contributing
 
-Thanks for your interest in contributing! This document contains `nats-io/nats.go` specific contributing details. If you are a first-time contributor, please refer to the general [NATS Contributor Guide](https://nats.io/contributing/) to get a comprehensive overview of contributing to the NATS project.
+Thanks for your interest in contributing! This document contains `nats-io/nats.go` specific contributing details. If you
```

Comment:
fixes to the contributing doc could be a separate commit in a different by itself, might be good to take it out of this PR then rebase